PLAN.md

```diff
@@ -0,0 +1,49 @@
+# Plan for chitter challenge
```

Comment:
I like this document - I didn't see a reference to it in README. I like how you've documented your project... I need to do this